### PR TITLE
adding keyboard: houndstooth

### PR DESCRIPTION
**Pull request by worldspawn00**
## Description

adding keyboard: houndstooth
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/wsk/houndstooth/config.h [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
```
Comment (drashna) [inline keyboards/wsk/houndstooth/config.h]: This is using the default (both shifts), but it is using the old noation, which may not be as accurate. 

You can safely remove this block.

File: keyboards/wsk/houndstooth/config.h [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+)
+
+/* prevent stuck modifiers */
+#define PREVENT_STUCK_MODIFIERS
```
Comment (drashna) [inline keyboards/wsk/houndstooth/config.h]: This is the default behavior.

```suggestion
```

File: keyboards/wsk/houndstooth/config.h [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+#define RGB_DI_PIN F1
+//#ifdef RGB_DI_PIN
+//#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 0
```
Comment (drashna) [inline keyboards/wsk/houndstooth/config.h]: 0? 😆 

May want to leave it undefined then. 

```suggestion
// #define RGBLED_NUM 0
```

File: keyboards/wsk/houndstooth/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+	[0] = LAYOUT(
+		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, 
+		KC_ESC, KC_A, KC_S, KC_D, KC_F, KC_G, 
+		KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, 
+		KC_LCTL, KC_LALT, KC_LGUI, MO(1), KC_SPC, KC_SPC,
+		
+		KC_Y, KC_U, KC_I, KC_O, KC_P, KC_BSPC, 
+		KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, 
+		KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_ENT, 
+		KC_SPC, KC_SPC, KC_LEFT, KC_DOWN, KC_UP, KC_RGHT),
```
Comment (drashna) [inline keyboards/wsk/houndstooth/keymaps/default/keymap.c]: /me cries

File: keyboards/wsk/houndstooth/houndstooth.h [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+#define KB_H
+
+#include "quantum.h"
+
+#define LAYOUT( \
+	K00, K01, K02, K03, K04, K05, K40, K41, K42, K43, K44, K45,\
+	K10, K11, K12, K13, K14, K15, K50, K51, K52, K53, K54, K55,\
+	K20, K21, K22, K23, K24, K25, K60, K61, K62, K63, K64, K65,\
+	K30, K31, K32, K33, K34, K35, K70, K71, K72, K73, K74, K75 \
+) { \
```
Comment (drashna) [inline keyboards/wsk/houndstooth/houndstooth.h]: If the layout has to be split up the way it is below, then you should reformat that block like this, so you can use a normal layout, not the split in half thing. 

```suggestion
#define LAYOUT( \
	K00, K01, K02, K03, K04, K05, K20, K21, K22, K23, K24, K25, \
	K40, K41, K42, K43, K44, K45, K60, K61, K62, K63, K64, K65, \
	K10, K11, K12, K13, K14, K15, K30, K31, K32, K33, K34, K35, \
	K50, K51, K52, K53, K54, K55, K70, K71, K72, K73, K74, K75 \
) { \
```

File: keyboards/wsk/houndstooth/readme.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# Houndstooth
+
+![Houndstooth](https://i.imgur.com/YkD4GNG.png)  
+
+The Houndstooth is a 12 column split ortho 40% keyboard.  
+
+Keyboard Maintainer: [worldspawn00](https://github.com/worldspawn00/)
```
Comment (fauxpark) [inline keyboards/wsk/houndstooth/readme.md]:
```suggestion
* Keyboard Maintainer: [worldspawn00](https://github.com/worldspawn00/)
```

File: keyboards/wsk/houndstooth/readme.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Houndstooth
+
+![Houndstooth](https://i.imgur.com/YkD4GNG.png)  
+
+The Houndstooth is a 12 column split ortho 40% keyboard.  
+
+Keyboard Maintainer: [worldspawn00](https://github.com/worldspawn00/)  
+Hardware Supported: The Houndstooth PCB and kit.
```
Comment (fauxpark) [inline keyboards/wsk/houndstooth/readme.md]:
```suggestion
* Hardware Supported: The Houndstooth PCB and kit.
```

File: keyboards/wsk/houndstooth/readme.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# Houndstooth
+
+![Houndstooth](https://i.imgur.com/YkD4GNG.png)  
+
+The Houndstooth is a 12 column split ortho 40% keyboard.  
+
+Keyboard Maintainer: [worldspawn00](https://github.com/worldspawn00/)  
+Hardware Supported: The Houndstooth PCB and kit.  
+Hardware Availability: [PCB & Kit](https://geekhack.org/index.php?topic=101374)
```
Comment (fauxpark) [inline keyboards/wsk/houndstooth/readme.md]:
```suggestion
* Hardware Availability: [PCB & Kit](https://geekhack.org/index.php?topic=101374)
```

File: keyboards/wsk/houndstooth/readme.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+# Houndstooth
+
+![Houndstooth](https://i.imgur.com/YkD4GNG.png)
```
Comment (fauxpark) [inline keyboards/wsk/houndstooth/readme.md]:
```suggestion
![Houndstooth](https://i.imgur.com/YkD4GNG.png)
```

File: keyboards/wsk/houndstooth/readme.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+# Houndstooth
+
+![Houndstooth](https://i.imgur.com/YkD4GNG.png)  
+
+The Houndstooth is a 12 column split ortho 40% keyboard.
```
Comment (fauxpark) [inline keyboards/wsk/houndstooth/readme.md]:
```suggestion
The Houndstooth is a 12 column split ortho 40% keyboard.
```

File: keyboards/wsk/houndstooth/rules.mk [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE ?= yes	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE ?= yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE ?= yes	# Audio control and System control(+450)
+CONSOLE_ENABLE ?= no	# Console for debug(+400)
+COMMAND_ENABLE ?= no    # Commands for debug and configuration
```
Comment (ridingqwerty) [inline keyboards/wsk/houndstooth/rules.mk]:
```suggestion
COMMAND_ENABLE = no    # Commands for debug and configuration
```

File: keyboards/wsk/houndstooth/rules.mk [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+SLEEP_LED_ENABLE ?= no  # Breathing sleep LED during USB suspend
```
Comment (ridingqwerty) [inline keyboards/wsk/houndstooth/rules.mk]:
```suggestion
SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
```

File: keyboards/wsk/houndstooth/rules.mk [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+NKRO_ENABLE ?= yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (ridingqwerty) [inline keyboards/wsk/houndstooth/rules.mk]:
```suggestion
NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```

File: keyboards/wsk/houndstooth/rules.mk [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+BACKLIGHT_ENABLE ?= no  # Enable keyboard backlight functionality
```
Comment (ridingqwerty) [inline keyboards/wsk/houndstooth/rules.mk]:
```suggestion
BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
```

File: keyboards/wsk/houndstooth/rules.mk [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+AUDIO_ENABLE ?= no
```
Comment (ridingqwerty) [inline keyboards/wsk/houndstooth/rules.mk]:
```suggestion
AUDIO_ENABLE = no
```

File: keyboards/wsk/houndstooth/rules.mk [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+RGBLIGHT_ENABLE ?= yes
```
Comment (ridingqwerty) [inline keyboards/wsk/houndstooth/rules.mk]:
```suggestion
RGBLIGHT_ENABLE = yes
```

**Comment by ridingqwerty** [comment]
Committed collaborator suggestions, requesting another look from @drashna and @fauxpark

**Comment by ridingqwerty** [comment]
Taking another look as Travis is choking on `RGBLED_NUM` and a missing comma in keymap.c

**Comment by ridingqwerty** [comment]
Looking better, `RGBLIGHT_ENABLE` should have been set to `no`

**Pull request merged**